### PR TITLE
Address Query

### PR DESCRIPTION
**Pull request by shanyizhang**
API: /customers/<int:customer_id>/addresses/<int:address_id>

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -228,6 +228,21 @@ def deactivate_customers(customer_id):
...
+### RETRIEVE AN ADDRESS FROM ACCOUNT
+### -----------------------------------------------------------
+@app.route('/customers/<int:customer_id>/addresses/<int:address_id>', methods=['GET'])
+def get_addresses(customer_id, address_id):
+    """
+    Get an Address
+    Just an address get returned
+    """
+    app.logger.info("Request to get an address with id: %s", address_id)
+    address = Address.find(address_id)
```
Comment (shcai97) [inline service/routes.py:242]: need to handle situation when address not found

File: tests/test_service.py [hunk 2]
```diff
@@ -426,3 +426,34 @@ def test_update_customer_not_found(self):
...
+                "zip_code": "100"
+            },
+            "active": True
+        }
+        resp = self.app.post(BASE_URL,
+                             json=body,
+                             content_type=CONTENT_TYPE_JSON)
+        data = resp.get_json()
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+        #self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
```
Comment (shcai97) [inline tests/test_service.py:454]: test case for address not found also needed

**Review by shcai97 — COMMENTED** [review]
Maybe we need to handle the situation that the address is not found like what we did in retrieving a customer.

**Comment by shanyizhang** [comment]
Good Catch. Now this situation when Address ID is not found gets handled.

**Pull request merged**